src/main/java/seedu/address/model/pair/Pair.java

```diff
@@ -0,0 +1,87 @@
+package seedu.address.model.pair;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.Objects;
+
+import seedu.address.model.person.Person;
+
+/**
+ * Represents a Pair of elderly and volunteer in the address book.
+ * Guarantees: details are present and not null, field values are validated, immutable.
+ */
+public class Pair {
```

Comment:
can you create an attribute for the `UUID`as well? maybe also extends from an interface, `Identifiable` that contains the basic getters and setters.

Reply:
I believe this is already done by heyi and shaun, as the NRIC class

Reply:
what's should be the current syntax for deleting a pair? are we specifying the pair or the two entities?

Reply:
If we use entities we would need to specify 2 different ids. If we specify a pair, we can use the id of the pair. Id of pair seems easier to use and quicker to type

Reply:
Discussed in meeting 4